src/pcm/SoxrResampler.cxx

```diff
@@ -80,20 +90,118 @@ soxr_parse_quality(const char *quality) noexcept
 	return SOXR_INVALID_RECIPE;
 }
 
+static unsigned
+SoxrParsePrecision(unsigned value) {
+	switch (value) {
+		case 16:
+		case 20:
+		case 24:
+		case 28:
+		case 32:
```

Comment:
Bad indent

Reply:
The preview in github shows more indent then it really is in the file itself. Please look at the raw file.
The `case` has two times the indent as for `switch`.

Reply:
>  The `case` has two times the indent as for `switch`.

That's the problem. It should have the same indent.
Coding style is a matter of taste, of course - and your indentation isn't absolutely wrong, just doesn't match the coding style used by the rest of MPD.